Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -720,6 +722,68 @@ private void SetOpenResultModifiers()
             OpenResultCommandModifiers = _settings.OpenResultModifiers ?? DefaultOpenResultModifiers;
         }
 
+        private static string GetActiveExplorerPath()
+        {
+            // get the active window
+            IntPtr handle = GetForegroundWindow();
+
+            // Required ref: SHDocVw (Microsoft Internet Controls COM Object) - C:\Windows\system32\ShDocVw.dll
+            ShellWindows shellWindows = new SHDocVw.ShellWindows();
+
+            // loop through all windows
+            foreach (var window in shellWindows)
+            {
+                if (window is not SHDocVw.InternetExplorer)
+                {
+                    continue;
+                }
+
+                var explorerWindow = (SHDocVw.InternetExplorer)window;
+                // match active window
+                if (explorerWindow.HWND == (int)handle)
+                {
+                    // Required ref: Shell32 - C:\Windows\system32\Shell32.dll
+                    var shellWindow = explorerWindow.Document as Shell32.IShellFolderViewDual2;
+
+                    // will be null if you are in Internet Explorer for example
+                    if (shellWindow != null)
+                    {
+                        // Item without an index returns the current object
+                        var currentFolder = shellWindow.Folder.Items().Item();
```

Comment:
Maybe you can fire a pr based on this one, and modify code there. @Garulf will have access to modify your pr so it would be easier to review and edit the code.